numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1788,6 +1791,24 @@ unpack_bits(PyObject *input, int axis, PyObject *count_obj, char order)
         goto fail;
     }
 
+    /*
+     * setup lookup table under GIL, 256 8 byte blocks representing 8 bits
+     * expanded to 1/0 bytes
+     */
+    if (unpack_init == 0) {
+        npy_uint64 j;
+        for (j=0; j < 256; j++) {
+            npy_uint64 k;
```

Comment:
This can be a uint8, right? Or perhaps a intp, since it's an array index. Not that it really matters...